deny.toml

```diff
@@ -7,7 +7,7 @@ confidence-threshold = 0.93
 # Commented license types are allowed but not currently used
 allow = [
     "Apache-2.0",
-    # "BSD-2-Clause",
```

Comment:
Is this needed?

Reply:
Yes. Due to the absence of Cargo.lock, make build fails. This commit is also a part of another PR - #48